app/services/albums.js

```diff
@@ -0,0 +1,24 @@
+const request = require('../helpers/request');
+const logger = require('../logger');
+
+const JPH_API_PATH = process.env.JSON_PLACE_HOLDER_API_PATH;
```

Comment:
Please check this folder: `al-express-js/config/`, there is a way to handle all environment variables

Reply:
Another 'layer 8'. How could I've forgotten, sorry for that.

Reply:
Fixed it!